### ISSUE TITLE
The meaning of a picture created during training

### ISSUE THREAD

**Issue by yishanyang** [panakino/FBPConvNet#1]
Hi,
when I run the code to train the network, a picture named figure 1 was created. Could you explain more meaning about that?

Danyang Li

**Comment by panakino** [comment]
If you are seeing the two graphs, the meaning of left graph is cost functional values along with epochs within training set(blue) and validation set(red). The cost function is a  target value to be minimized in regression type neural network. Simply speaking, it shows how training or validation data set is close to label (groundtruth, what we want to be in final reconstruction).
